src/components/v5/common/Fields/InputBase/InputBase.tsx

```diff
@@ -16,13 +18,15 @@ const InputBase = React.forwardRef<HTMLInputElement, InputBaseProps>(
       state,
       message,
       prefix,
+      value,
       suffix,
       mode = 'primary',
       disabled,
       stateClassNames: stateClassNamesProp,
       autoWidth = false,
       label,
       id: idProp,
+      maxLength,
```

Comment:
do we want to add `maxLength` to the input too?

Reply:
@bassgeta  yes, it's necessary to display message error

Reply:
sorry, misunderstaning, I mean that if we could add the maxLength to the `<input />` element as an HTML property too

Reply:
@bassgeta  when we add `maxLength` also to <input /> element then information about min/max characters will appears only when value in the input will be empty. So the question is should this work in that way?

Reply:
I see, well I don't know what the UX should be like, but it's pretty normal to show a character counter underneath fields with a maximum length, and disabling further input when you hit it. If that is not how our UX works, then we can not add the `maxLength` attribute onto the HTML tags

Reply:
yes I aggree with you.
but regarding to this task and the design it looks like user will able to add more characters so as I understand we can't add `maxLength` attribute onto the HTML tags 
https://tw.auroracreation.com/app/tasks/15687546
<img width="458" alt="Screen Shot 2023-12-13 at 13 52 04" src="https://github.com/JoinColony/colonyCDapp/assets/119587095/6ae26f42-8d43-4136-a848-d83f09dad530">

Reply:
alright if we don't show the max length before it's an error, then this behaviour is correct, I'll resolve both comments

Reply:
@bassgeta are we agree to merge this pull request?